src/main/java/seedu/duke/commands/DeleteCommand.java

```diff
@@ -85,4 +92,25 @@ public void deleteTaskFromModule(Module targetModule) throws ModHappyException {
         TaskList taskList = targetModule.getTaskList();
         result = String.format(DELETE_MESSAGE, taskList.removeTask(taskIndex));
     }
+
+    /**
+     * Gets confirmation from user to delete given module.
+     *
+     * @param module Module to be deleted.
+     * @return Returns true if user input is "yes", false if "no".
+     * @throws ParseException Throws an exception if user input is not "yes" or "no".
+     */
+    public Boolean getUserConfirmation(Module module) throws ParseException {
+        Scanner scanner = new Scanner(System.in);
```

Comment:
Im not sure if instantiating a new TextUi object here would be better to be consistent with the UI component. I think integrating with Main might be harder and non-trivial here so if we do change this, then we will need to update the DG